VideoRecStation/src/maindialog.cpp

```diff
@@ -148,6 +149,7 @@ void MainDialog::cleanVideoDialog(unsigned int idx)
     settings.videoGains[idx] = videoDialogs[idx]->findChild<QSlider*>("gainSlider")->value();
     settings.videoUVs[idx] = videoDialogs[idx]->findChild<QSlider*>("uvSlider")->value();
     settings.videoVRs[idx] = videoDialogs[idx]->findChild<QSlider*>("vrSlider")->value();
+    settings.videoLimits[idx] = videoDialogs[idx]->findChild<QCheckBox*>("ldsBox")->isChecked();
```

Comment:
You can search for instances of `ldsBox` to find the lines that were changed to add that option.